ethereum/client.go

```diff
@@ -1308,11 +1347,29 @@ func (ec *Client) Call(
 			return nil, fmt.Errorf("%w: %s", ErrCallParametersInvalid, err.Error())
 		}
 
-		if len(input.ContractAddress) == 0 || len(input.Data) == 0 {
-			return nil, fmt.Errorf("%w:contract_address or data missing from params", ErrCallParametersInvalid)
+		if len(input.To) == 0 || len(input.Data) == 0 {
+			return nil, fmt.Errorf("%w:to address or data missing from params", ErrCallParametersInvalid)
+		}
+
+		resp, err := ec.contractCall(ctx, input.BlockIndex, input.BlockHash, input.To, input.Data)
+		if err != nil {
+			return nil, err
+		}
+
+		return &RosettaTypes.CallResponse{
+			Result: resp,
+		}, nil
+	case "eth_estimateGas":
+		var input GetCallInput
+		if err := RosettaTypes.UnmarshalMap(request.Parameters, &input); err != nil {
+			return nil, fmt.Errorf("%w: %s", ErrCallParametersInvalid, err.Error())
+		}
+
+		if len(input.To) == 0 || len(input.Data) == 0 {
+			return nil, fmt.Errorf("%w:to address or data missing from params", ErrCallParametersInvalid)
 		}
```

Comment:
You might want to move this validation to its own function since `eth_call` also uses it.